duckiebots:
  our_duckie:
    start_pose: [1.0, 0.0, 0] # [x,y, theta]
    velocity: 1.0
    radius: 0.25
    type: "our_duckie"
    angle_change_limit: 0.2 # rad
  unstable_speed_duckie:
    start_pose: [-0.5, 10.0, 3.14] # [x,y, theta]
    velocity: 0.4
    radius: 0.6
    type: "unstable_speed_duckie" 
    angle_change_limit: 0.2 # rad
    max_acceleration: 2
    max_velocity: 1
    min_velocity: 0.1
  constant_speed_duckie:
    start_pose: [-0.5, 10000.0, 3.14] # [x,y, theta]
    velocity: 0
    radius: 0.6
    type: "constant_speed_duckie" 
    angle_change_limit: 0.2 # rad
    max_acceleration: 0
    max_velocity: 0
    min_velocity: 0
